internal/wasm/sdk/opa/opa.go

```diff
@@ -44,7 +44,7 @@ type EntrypointID int32
 // initialized before invoking the Eval.
 func New() *OPA {
 	opa := &OPA{
-		memoryMinPages: 2,
+		memoryMinPages: 16,
```

Comment:
Should we just take `len(policyBytes)` and round to the nearest page size multiple?